otx/cli/utils/config.py

```diff
@@ -30,3 +34,38 @@ def override_parameters(overrides, parameters):
                 parameters[k] = val
             else:
                 raise ValueError(f'The "{k}" is not in allowed_keys: {allowed_keys}')
+
+
+def configure_dataset(args, train=False):
+    """Configure dataset args."""
+
+    # Create instances of Task, ConfigurableParameters and Dataset.
+    data_subset_format = {"ann-files": None, "data-roots": None}
+    data_config = {"data": {subset: data_subset_format.copy() for subset in ("train", "val", "test")}}
+    data_config["data"]["unlabeled"] = {"file-list": None, "data-roots": None}
+    if os.path.exists(args.data):
+        with open(args.data, "r", encoding="UTF-8") as stream:
+            data_config = yaml.safe_load(stream)
+        stream.close()
+        if train:
+            args.save_model_to = "./models"
```

Comment:
Can't we get the custom path from save_model_to arg? It looks like it always saves train ckpt in './models'. Maybe I can miss something so could you check? And I'm curious which path models and logs are if data.yaml is not loaded and just initiated 'otx train' with default path and custom args. In that case -> I guess it's gonna be in 'training_extension/models/' and 'training_extension/logs' ?

Reply:
This looks like my mistake. thanks for the nice comment! By default, it targets `otx-workspace/models` and `workspace/logs`, but it seems most appropriate to use those paths if there are commands. I will correct this.

Reply:
Fixed it!